interactions/context.py

```diff
@@ -280,7 +280,10 @@ async def edit(
 
         if isinstance(components, ActionRow):
             _components[0]["components"] = [component._json for component in components.components]
-        elif isinstance(components, (Button, SelectMenu)):
```

Comment:
What is the relation of this code here relevant to new functions for models?

Reply:
No relation, I encountered it when making the edit, I'll put it into the other PR